spec/builtin/dns/suffix_enum_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'ronin/recon/builtin/dns/suffix_enum'
+
+RSpec.describe Ronin::Recon::DNS::SuffixEnum do
+  let(:fixtures_dir) { File.join(__dir__,'..', '..','fixtures') }
+
+  describe '#process' do
+    context 'when there is a domain with a different suffix' do
+      let(:domain)      { Ronin::Recon::Values::Domain.new('example.com') }
+      let(:suffix_path) { File.join(fixtures_dir, 'builtin', 'dns', 'suffix_enum', 'public_domains.dat') }
```

Comment:
You could move this fixture file into a `spec/builtin/dns/fixtures/` directory, so the file is closer to the other DNS spec files.